src/Tools/dotnet-monitor/Egress/EgressService.cs

```diff
@@ -51,6 +51,24 @@ public async Task<EgressResult> EgressAsync(string providerName, Func<Stream, Ca
             return new EgressResult(result.ArtifactPath);
         }
 
+        public async Task<EgressResult> ValidateProviderAsync(string providerName, CancellationToken token)
```

Comment:
```suggestion
        public async Task ValidateProviderAsync(string providerName, CancellationToken token)
```

Callers shouldn't care about the result and the result is fairly uninformative in this case since it would always be successful with an empty artifact path. Just throw on failure and return void on success.